src/api/stats/getTvl.js

```diff
@@ -1,17 +1,11 @@
 const getChainTvl = require('./getChainTvl.js');
 
-const {
-  AVAX_CHAIN_ID,
-  AVAX_VAULTS_ENDPOINT,
+const { TELOS_CHAIN_ID, TELOS_VAULTS_ENDPOINT } = require('../../constants');
```

Comment:
Nit: same thing here with the line breaks

Reply:
Talked on Discord, we will update the prettier config to account for styling preferences.